src/main/menus/app.js

```diff
@@ -210,22 +210,14 @@ function createTemplate(mainWindow, config, isDev) {
     submenu: [{
       label: 'Back',
       accelerator: process.platform === 'darwin' ? 'Cmd+[' : 'Alt+Left',
-      click: (item, focusedWindow) => {
-        if (focusedWindow === mainWindow) {
-          mainWindow.webContents.send('go-back');
-        } else if (focusedWindow.webContents.canGoBack()) {
-          focusedWindow.goBack();
-        }
+      click: () => {
+        mainWindow.webContents.send('go-back');
```

Comment:
If I hit back on the popup, wouldn't this make the mainWindow to go back? I feel that is not what the user experience should feel like.
Wouldn't it be better to leave this as `if (focusedWindow === mainwindow) do nothing`? (Or even better if the button is not there, but I understand that is not possible / way more complicated)

Reply:
It's a menu shortcut, so we can't hide the button. We could do nothing on `focusedWindow !== mainWindow` I suppose, but I wanted the users to still be able to go back/forward with the main window. I'm 0/5 on this.

Reply:
Yes, I meant `focusedWindow !== mainWindow`. Nevertheless if this is the desired behaviour, I have no more complains.